src/commonConsts/UserID.ts

```diff
@@ -0,0 +1 @@
+export const USER_ID = 102;
```

Comment:
here I mean it can be a folder `consts`, and inside a file `commonConts.ts`

there is no need to create a name file for keepink only one consts:
`USER_ID.ts` name shows that it have only user id